functional-tests/src/lib.rs

```diff
@@ -410,6 +446,43 @@ b: ba"#
         assert_eq!(output.stdout, data);
     }
 
+    #[test]
+    fn roundtrip_kms_encryption_context() {
+        let file_path = prepare_temp_file("test_roundtrip_kms_encryption_context.json",
+                                          b"{
+    \"foo\": 2,
+    \"bar\": \"baz\"
+}");
+
+	let kms_arn = match env::var(KMS_KEY) {
```

Comment:
can this block be replaced with `env::var(KMS_KEY).expect(..)` ?

Reply:
Using `.expect()` when it was panicking makes sense, would I still want to use it with the change I made related to just ignoring the test if the env var is unset?

Reply:
No, since `.expect()` will panic if it's `None`.